infra/nnfw/python/pack.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# package
+python3 setup.py sdist bdist_wheel
```

Comment:
`pack.sh` does 2 different tasks: `packaging` and `publishing`. I think they should be in different script.

Someone may want to make a package, but don't want to publish.

Reply:
`pack.sh` is written for doing the package and publishing easily.
I think there are two options:
1) Change `pack.sh`  In the form of receiving options from the user 
     ex. for packaging, use `sh pack.sh -p` / for uploading, use `sh pack.sh -u`
2) Delete `pack.sh` shell script and provide information for each command at `README.md`

Reply:
For this PR, both work for me. Please choose what you feel convinient. I will get opinion frome others and update if necessary.